src/_check.sh

```diff
@@ -108,7 +108,7 @@ function _check_os_supported_flavor_and_release() {
 
 # Check for some base required packages to even validate the system
 function _check_base_required_packages() {
-  requiredCommands=('jq' 'lsb_release' 'wget' 'bc')
+  requiredCommands=('jq' 'lsb_release' 'wget' 'bc' 'docker')
```

Comment:
The utility itself handles docker installation, so ideally shouldn't depend on docker.

Reply:
Totally get the pre-install portion. This isn't complete and we're trying to get it to validate that `docker compose` vs `docker-compose` is working in the environment. That part is incomplete in this PR and a WIP